### PR TITLE
Allow mass-unassigning of issues

### PR DESCRIPTION
**Pull request by rspeicher**
Fixes https://gitlab.com/gitlab-org/gitlab-ce/issues/867

Also renames the spec file to correctly match its alternate file's name.

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://tsigo-rs-issue-867.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://tsigo-rs-issue-867.c68bfdc4cac4c32c41e1a922e73062ea.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: spec/services/issues/bulk_update_service_spec.rb [hunk 1]
```diff
@@ -84,6 +84,16 @@
...
+    it 'allows mass-unassigning' do
+      @project.issues.first.update_attribute(:assignee, @new_assignee)
+      expect(@project.issues.first.assignee).not_to be_nil
+
+      @params[:update][:assignee_id] = ''
+
+      Issues::BulkUpdateService.new(@project, @user, @params).execute
+      expect(@project.issues.first.assignee).to be_nil
+    end
+
```
Comment (tsigo) [inline spec/services/issues/bulk_update_service_spec.rb]: Matching existing style.

**Comment by tsigo** [comment]
Closing, has unwanted side-effects.

**Comment by tsigo** [comment]
Re-opened with a proper fix, #8869

**Pull request closed without merging**